CHANGELOG.md

```diff
@@ -1,14 +1,25 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+## [v2.29.5] 2021-11-29
+
+### Changed
+
+- Whitelisting updated
+
 ## [v2.29.4] 2021-11-17
+
 ### Changed
+
 - Whitelisting updated
 
 ## [v2.29.3] 2021-11-15
+
 ### Changed
+
```

Comment:
Some auto-formatting inserted the empty lines?

Reply:
fixed